templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends '@W3CWebsiteTemplates/base.html.twig' %}
+
+{% block main %}
+    <h1>{% block title %}{{ page.title|default('W3C - Error: {{ status_code }} - {{ status_text }}') }}{% endblock %}</h1>
+    <p>The server returned a "{{ status_code }} {{ status_text }}" error.</p>
```

Comment:
This sentence should be localized.

Reply:
addressed in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/2892c3eccf727bf9f955329c726187c634566c92